spec/models/bucket_list_spec.rb

```diff
@@ -2,12 +2,15 @@
 
 RSpec.describe BucketList, type: :model do
   subject { FactoryGirl.build(:bucket_list) }
-  # it { should have_many(:users) }
-  # it { should have_many(:destinations) }
+  it { should belong_to(:user) }
 
   it { should have_valid(:title).when('New Mexico', 'South East Asia with Kam') }
   it { should_not have_valid(:title).when(nil, '') }
   it { should validate_uniqueness_of(:title) }
 
   it { should validate_inclusion_of(:is_public).in_array([true, false]) }
+
+  it { should have_valid(:user_id).when(1, 10) }
+  it { should_not have_valid(:user_id).when(nil, '') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.